politeiad/cmd/politeia/politeia.go

```diff
@@ -58,12 +68,20 @@ func usage() {
 	fmt.Fprintf(os.Stderr, "  inventory         - Inventory records "+
 		"<vetted count> <branches count>\n")
 	fmt.Fprintf(os.Stderr, "  new               - Create new record "+
-		"<name> <filename>...\n")
+		"[mdid:metadata]... <filename>...\n")
```

Comment:
This parameter format doesn't make it clear that it should be `metadata<number>:<metadata payload>`.

Reply:
Tried to document it a bit better.